CHANGELOG.md

```diff
@@ -2,6 +2,100 @@
+- **Maintenance-Gems:** Bump rubocop from 1.56.4 to 1.61.0 [\#5406](https://github.com/consuldemocracy/consuldemocracy/pull/5406)
+- **Maintenance-Gems:** Bump dalli from 3.2.6 to 3.2.8 [\#5402](https://github.com/consuldemocracy/consuldemocracy/pull/5402)
+- **Maintenance-Gems:** Bump rubocop-performance from 1.19.1 to 1.20.2 [\#5384](https://github.com/consuldemocracy/consuldemocracy/pull/5384)
+- **Maintenance-Gems:** Bump audited from 5.4.0 to 5.4.3 [\#5382](https://github.com/consuldemocracy/consuldemocracy/pull/5382)
+- **Maintenance-Gems:** Bump autoprefixer-rails from 10.2.5.1 to 10.4.16.0 [\#5346](https://github.com/consuldemocracy/consuldemocracy/pull/5346)
+- **Maintenance-Gems:** Bump airbrake from 13.0.3 to 13.0.4 [\#5343](https://github.com/consuldemocracy/consuldemocracy/pull/5343)
+- **Maintenance-Gems:** Bump spring from 2.1.1 to 4.1.3 [\#5338](https://github.com/consuldemocracy/consuldemocracy/pull/5338)
+- **Maintenance-Gems:** Bump globalize from 6.2.1 to 6.3.0 [\#5308](https://github.com/consuldemocracy/consuldemocracy/pull/5308)
+- **Maintenance-Gems:** Bump pronto from 0.11.1 to 0.11.2 [\#5307](https://github.com/consuldemocracy/consuldemocracy/pull/5307)
+- **Maintenance-Gems:** Bump delayed_job_active_record from 4.1.7 to 4.1.8 [\#5300](https://github.com/consuldemocracy/consuldemocracy/pull/5300)
+- **Maintenance-Gems:** Bump acts-as-taggable-on from 9.0.1 to 10.0.0 [\#5298](https://github.com/consuldemocracy/consuldemocracy/pull/5298)
+- **Maintenance-Gems:** Bump responders from 3.1.0 to 3.1.1 [\#5296](https://github.com/consuldemocracy/consuldemocracy/pull/5296)
+- **Maintenance-Gems:** Bump devise from 4.9.2 to 4.9.3 [\#5293](https://github.com/consuldemocracy/consuldemocracy/pull/5293)
+- **Maintenance-Gems:** Bump paranoia from 2.6.2 to 2.6.3 [\#5292](https://github.com/consuldemocracy/consuldemocracy/pull/5292)
+- **Maintenance-Gems:** Bump graphql from 1.12.14 to 1.13.22 [\#4955](https://github.com/consuldemocracy/consuldemocracy/pull/4955)
+- **Maintenance-JavaScript:** Bump braces from 3.0.2 to 3.0.3 [\#5578](https://github.com/consuldemocracy/consuldemocracy/pull/5578)
+- **Maintenance-JavaScript:** Update Node.js from 18.18.2 to 18.20.3 [\#5543](https://github.com/consuldemocracy/consuldemocracy/pull/5543)
+- **Maintenance-JavaScript:** Use node packages to install Foundation [\#5446](https://github.com/consuldemocracy/consuldemocracy/pull/5446)
+- **Maintenance-JavaScript:** Replace rails-assets.org with Node.js packages [\#5423](https://github.com/consuldemocracy/consuldemocracy/pull/5423)
+- **Maintenance-Refactoring:** Rename Poll::Question::Answer to Poll::Question::Option [\#5537](https://github.com/consuldemocracy/consuldemocracy/pull/5537)
+- **Maintenance-SCSS:** Replace SassC/Libsass with Dart Sass [\#5477](https://github.com/consuldemocracy/consuldemocracy/pull/5477)
```

Comment:
Since we're adding a gem, and we don't have any other "Maintenance-SCSS" entries in the changelog, how about "Maintenance-Gems" for this one? :thinking: